docs/packaging/monitoring2.yml.md

```diff
@@ -0,0 +1,139 @@
+
+## systemd as an example
+
+Let's look at the `monitoring.yml` file for `systemd` as an example.
+
+The `systemd` directory in the Packages repository looks like this:
+
+```text
+systemd
+├── monitoring.yml
+├── package.yml
+├── pspec_x86_64.xml
+└── *lots of other files we can ignore*
+```
+
+The `monitoring.yml` looks like this:
+
+```yaml
+releases:
+  id: 205088
+  ignore:
+    # We only update to the n-1 stable release. So for now we're only interested in 252.x updates
+    - "253.*"
+  rss: [todo proper rss link]
+security:
+  cpe:
+    - vendor: systemd_project
+      product: systemd
+    - vendor: freedesktop
+      product: systemd
+  ignore:
+    - CVE-2022-55555
+```
+
+## "releases" fields
+
+| Field    | Type                        | Required ?              | Description                                                                                                                    |
+| -------- | --------------------------- | ----------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `id`     | integer                     | Yes                     | Anitya ID from [release-monitoring.org](https://release-monitoring.org/)                                                       |
+| `ignore` | list of regular expressions | No                      | List of regular expression matching versions to ignore. Include a comment explaining the ignored versions. Enclose in quotes   |
+| `rss`    | URL                         | No, strongly encouraged | URL for a releases RSS feed. If the only RSS feed you can find for a project is a general "news" feed, don't include the field |
+
+- Fields used to monitor for new versions
+
+### Finding the Anitya ID
+
+You can find the Anitya ID by searching [release-monitoring.org](https://release-monitoring.org/) by project name, then taking the ID out of the URL for the correct search result. For example, the correct `systemd` search result for us is `systemd-stable` with the URL [https://release-monitoring.org/project/205088/](https://release-monitoring.org/project/205088/), so we use `205088` in the `id field`
+
+## "security" fields
+
+| Field    | Type            | Required ? | Description                                                                         |
+| -------- | --------------- | ---------- | ----------------------------------------------------------------------------------- |
+| `cpe`    | list            | Yes        | List of `vendor:product` pairs from a full CPE name. CVEs for a given project may be published under more than one CPE; include more than one if that is likely.                            |
+| `ignore` | list of strings | No         | List of specific CVE identifiers which can be ignored. Todo: do we include reasons? |
```

Comment:
I think it would be good to clarify in which situations a CVE can be ignored.

Reply:
done